fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/mixin/biome/modification/MinecraftServerMixin.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.biome.modification;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.registry.DynamicRegistryManager;
+import net.minecraft.world.SaveProperties;
+import net.minecraft.world.level.LevelProperties;
+
+import net.fabricmc.fabric.impl.biome.modification.BiomeModificationImpl;
+
+@Mixin(MinecraftServer.class)
+public class MinecraftServerMixin {
+	@Shadow
+	private DynamicRegistryManager.Impl registryManager;
+
+	@Shadow
+	private SaveProperties saveProperties;
+
+	@Inject(method = "<init>", at = @At(value = "RETURN"))
+	private void finalizeWorldGen(CallbackInfo ci) {
```

Comment:
Does running the biome modifications here work with data pack biomes that are `/reload`ed?

Reply:
data pack biomes are not reloaded by reload as far as I know

Reply:
Afaik none of the dynamic registries are reloaded at all.

Reply:
TIL, this is all fine then